crane-cloud/backend
# Add follow state

# Description
 
Enable a requesting user to receive an attribute on whether they follow a project 

## Type of change




## Trello Ticket ID

https://trello.com/c/mFTNG8os

## How Can This Be Tested?

Try to  fetch  a user or project on this branch, you should be able to see the attributes mentioned

## Checklist:
